dts/riscv/starfive/jh7110-visionfive-v2.dtsi

```diff
@@ -0,0 +1,219 @@
+            next-level-cache = <&ccache>;
+            reg = <0x1>;
+            tlb-spilt;
+
+            riscv,isa = "rv64imafdcg";
+
+            cpu1_intc: interrupt-controller {
+                compatible = "riscv,cpu-intc";
+                #interrupt-cells = <1>;
+                interrupt-controller;
+            };
+        };
+
+        U74_2: cpu@2 {
+            compatible = "sifive,u74", "riscv";
+            device_type = "cpu";
+            mmu-type = "riscv,sv39";
+            d-cache-block-size = <64>;
+            d-cache-sets = <64>;
+            d-cache-size = <32768>;
+            d-tlb-sets = <1>;
+            d-tlb-size = <40>;
+            i-cache-block-size = <64>;
+            i-cache-sets = <64>;
+            i-cache-size = <32768>;
+            i-tlb-sets = <1>;
+            i-tlb-size = <40>;
+            next-level-cache = <&ccache>;
+            tlb-split;
+            reg = <0x2>;
+            riscv,isa = "rv64imafdcg";
+
+            cpu2_intc: interrupt-controller {
+                compatible = "riscv,cpu-intc";
+                #interrupt-cells = <1>;
+                interrupt-controller;
+            };
+        };
+
+        U74_3: cpu@3 {
+            compatible = "sifive,u74", "riscv";
+            device_type = "cpu";
+            mmu-type = "riscv,sv39";
+            d-cache-block-size = <64>;
+            d-cache-sets = <64>;
+            d-cache-size = <32768>;
+            d-tlb-sets = <1>;
+            d-tlb-size = <40>;
+            i-cache-block-size = <64>;
+            i-cache-sets = <64>;
+            i-cache-size = <32768>;
+            i-tlb-sets = <1>;
+            i-tlb-size = <40>;
+            next-level-cache = <&ccache>;
+            tlb-split;
+            reg = <0x3>;
+            riscv,isa = "rv64imafdcg";
+
+            cpu3_intc: interrupt-controller {
+                compatible = "riscv,cpu-intc";
+                #interrupt-cells = <1>;
+                interrupt-controller;
+            };
+        };
+
+        U74_4: cpu@4 {
+            compatible = "sifive,u74", "riscv";
+            device_type = "cpu";
+            mmu-type = "riscv,sv39";
+            d-cache-block-size = <64>;
+            d-cache-sets = <64>;
+            d-cache-size = <32768>;
+            d-tlb-sets = <1>;
+            d-tlb-size = <40>;
+            i-cache-block-size = <64>;
+            i-cache-sets = <64>;
+            i-cache-size = <32768>;
+            i-tlb-sets = <1>;
+            i-tlb-size = <40>;
+            next-level-cache = <&ccache>;
+            tlb-split;
+            reg = <0x4>;
+            riscv,isa = "rv64imafdcg";
+
+            cpu4_intc: interrupt-controller {
+                compatible = "riscv,cpu-intc";
+                #interrupt-cells = <1>;
+                interrupt-controller;
+            };
+        };
+    };
+
+    ram0: memory@8000000 {
+        device_type = "memory";
+        reg = <0x0 0x8000000 0x1 0x0>;
+    };
+
+    soc {
```

Comment:
Let's add a compatible string for the SoC node

Reply:
Added the `compatible` and `model` node to the `soc` node.